source/menu.cpp

```diff
@@ -586,8 +586,14 @@ void GenerateRandomizer() {
 
 	int ret = Playthrough::Playthrough_Init(finalHash);
 	if (ret < 0) {
-		printf("Error %d with fill. Press Select to exit.\n", ret);
-		return;
+    if(ret == -1) { //Failed to generate after 5 tries
```

Comment:
This looks a little wonky indentation-wise, were tabs and spaces intermixed on these lines?

Reply:
Indeed they were, not sure how that happened but it look fine on my IDE so I didn't notice- Thanks for pointing it out